project0API/Script-2.sql

```diff
@@ -0,0 +1,17 @@
+-- account table
+create table accounts(
+	account_id serial primary key,
+	account_title text unique not null,
+	account_balance int check(account_balance > 0)
+);
+
+
+
+-- user table
+create table users(
+	user_id serial primary key,
+	user_first_name text not null,
+	user_last_name text not null,
+	user_id_fk int references accounts(account_id)
```

Comment:
As @yantaozhu pointed out on Discord. The foreign key will need to be in the `accounts` table.

Reply:
Also the `user_id_fk` will reference the `user_id` like so...

```
user_id_fk int references users(user_id)
```

Reply:
For reference:

```SQL
create table users(
	user_id serial primary key,
	user_first_name text not null,
	user_last_name text not null
);

create table accounts(
	account_id serial primary key,
	account_title text not null,
	account_balance int,
	user_id_fk int references users(user_id)
);
```